production_services/settings.py

```diff
@@ -73,7 +77,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [os.path.join(BASE_DIR, 'ensembl_production/templates')],
+        'DIRS': [os.path.join(BASE_DIR, 'ensembl_production/templates'),os.path.join(BASE_DIR, 'ensembl_dbcopy/templates')],
```

Comment:
Since following statement is set at "True", both these statements should be obsolete since these are both "apps"

Reply:
Done in latest commit